fellowship/index.html

```diff
@@ -123,31 +123,31 @@ <h1 class="text-primary">The Impact Fellowship</h1>
 		<div class="d-none d-lg-inline" style="position: fixed; height:30%; bottom:0; z-index: -2; width:100%; background-color:white;"></div>
 
 
-		<section class="mt-5" style="position:relative" >
-		<img alt="Bolt" class="accent d-none d-md-inline" style="width:100px; position:absolute; right: 100px; top: -100px;" src="../assets/bolt.svg" />
-		<img alt="Plus" class="accent d-none d-md-inline" style="width:120px; position:absolute; left: 250px; top: 500px;" src="../assets/plus.svg" />
-		<div class="container-fluid">
-			<div class="row">
-				<div class="col-0 col-md-4"></div>
-				<div class="col-12 col-md-4">
-				<h2>About the Fellowship</h2>
-				<p class="lead">
-					The Impact Fellowship is a 2-week gathering of talented and passionate computer science students who want to use their skills for social good!
-				</p>
-				</div>
-				<div class="col-12 col-md-3 mt-5 text-center">
-					<img alt="Abou the Fellowship" width="350px" src="../assets/illustrations/about_the_fellowship.svg" />
+		<section style="position:relative" >
+			<img alt="Bolt" class="accent d-none d-md-inline" style="width:100px; position:absolute; right: 100px; top: -100px;" src="../assets/bolt.svg" />
+			<img alt="Plus" class="accent d-none d-md-inline" style="width:120px; position:absolute; left: 250px; top: 500px;" src="../assets/plus.svg" />
+			<div class="container-fluid">
+				<div class="row">
+					<div class="col-0 col-md-4"></div>
+					<div class="col-12 col-md-4">
+					<h2>About the Fellowship</h2>
+					<p class="lead">
+						The Impact Fellowship is a 2-week gathering of talented and passionate computer science students who want to use their skills for social good!
+					</p>
+					</div>
+					<div class="col-12 col-md-3 mt-5 text-center">
+						<img alt="Abou the Fellowship" width="350px" src="../assets/illustrations/about_the_fellowship.svg" />
```

Comment:
not part of this pr but typo on "about the fellowship"

Reply:
Good catch